R/expect_lint.R

```diff
@@ -158,3 +155,13 @@ expect_lint_free <- function(...) {
 
   invisible(result)
 }
+
+# Helper function to check if testthat is installed.
+require_testthat <- function(name) {
+  if (!requireNamespace("testthat", quietly = TRUE)) {
+    stop( # nocov start
+      name, "is designed to work within the 'testthat' testing framework, but 'testthat' is not installed.",
```

Comment:
Doesn't `stop()` use `sep = ""`, butchering the error message like this?

Reply:
You are correct. Added a space.